lib/exception_handling/logging_methods.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'active_support/concern'
+require 'active_support/core_ext/module/delegation.rb'
+
+module ExceptionHandling
+  module LoggingMethods # included on models and controllers
+    extend ActiveSupport::Concern
+
+    protected
+
+    delegate :log_error_rack, :log_warning, :log_info, :log_debug, :escalate_error, :escalate_warning, :ensure_escalation, :alert_warning, to: ExceptionHandling
+
+    # TODO: delegate log_error as well
+    def log_error(exception_or_string, exception_context = '')
+      controller = self if respond_to?(:request) && respond_to?(:session)
```

Comment:
@madelynrr I think we can move this `controller =` part over to `::Methods`. It shouldn't be needed here in `LoggingMethods`, so I don't think any tests will fail.

Reply:
Removed, and delegated log_error  to exception_handling.rb.